samples/Mvc/Mvc.Server/Controllers/AuthenticationController.cs

```diff
@@ -13,33 +15,53 @@ public class AuthenticationController : Controller {
             ViewBag.ReturnUrl = returnUrl;
 
             // Note: in a real world application, you'd probably prefer creating a specific view model.
-            return View("SignIn", Context.GetExternalProviders());
+            return View("SignIn", new AuthenticationModel() {AuthenticationDescriptions = Context.GetExternalProviders()});
         }
 
         [HttpPost("~/signin")]
-        public ActionResult SignIn(string provider, string returnUrl) {
+        public ActionResult SignIn(AuthenticationModel  model) {
```

Comment:
nit: spacing.

I wonder if we shouldn't create another `SignIn` action for local sign and use 2 different HTML forms in the client.
Also, the view model shouldn't be necessary.